proto/pairing/badge_used_cu.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+package lavanet.lava.pairing;
+
+option go_package = "github.com/lavanet/lava/x/pairing/types";
+
+message BadgeUsedCu {
+  bytes badgeUsedCuMapKey = 1; 
```

Comment:
"Map" is an implementation detail.  just "badge_used_cu_key" suffices.

Reply:
done